source/blog/2019-01-04-the-ember-times-issue-79.md

```diff
@@ -1,39 +1,24 @@
 ---
 title: The Ember Times - Issue No. 79
-author: Chris Ng, Jessica Jordan, Kenneth Larsen, Michael Schinis, the crowd
+author: Chris Ng, Jessica Jordan, Kenneth Larsen, Michael Schinis
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/01/04-the-ember-times-issue-79.html"
 responsive: true
 ---
 
 Blein Vie Noa, Emberistas! 🐹🎆🎇
 
-The new year starts off with loads of **new RFCs**! Read more about **extending** the **Ember Inspector** 👓🐹, and suggested **deprecations** of **Route render** methods and selected **ApplicationController properties** �! We also have a new RFC for a brand new look of emberjs.com! And a new beta release of **ember-cli-babel**!
-
----
-
-## [SECTION TITLE](#section-url)
-
+The new year starts off with loads of **new RFCs**! Read more about **extending** the **Ember Inspector** 👓🐹, and suggested **deprecations** of **Route render** methods and selected **ApplicationController properties** �! We also have a new RFC for a brand new look of emberjs.com, **performance improvements** 🏎 for the API Docs **search**, a new beta release of **ember-cli-babel** and an advanced testing **exam** for you!
```

Comment:
`and suggested **deprecations**` I think the `and` in unneeded here :)

Also the emoji after `**ApplicationController properties**` isn't showing up for me